README.md

```diff
@@ -190,6 +190,13 @@ Currently we only support one material type per mesh (see [known limitations](#k
 
 ## Release Notes
 
+### v2.3.0 (2024-01-16)
+Required Cityengine version: 2023.1 or older.
+For commercial work, a [license](https://www.esri.com/cityengine) for CityEngine 2022.0 or later is required.
```

Comment:
I think this should say "2023.1". Our license blurb says: "Commercial use requires at least one commercial license of the **latest** CityEngine version installed in the organization."

Reply:
fixed